### DISCUSSION TITLE
Where are the Incan ruins of Machu Picchu located?

### DISCUSSION THREAD

**Discussion by ParanoidUser** [ParanoidUser/yolo#471] [Galaxy Trivia] [answered]
**Bonus:** When was Machu Picchu founded?

**Comment by peterbaccaro** [comment, marked as answer]
The Inca ruins of Machu Picchu are tucked away in the rocky countryside northwest of **Cuzco, Peru**.
Machu Picchu was believed to have been built in the 1450s however a 2021 study led by Burger used radiocarbon dating AMS to reveal that Machu Picchu may have been occupied from around 1420–1530 AD. Construction appears to date from two great Inca rulers, Pachacutec Inca Yupanqui (1438–1471) and Túpac Inca Yupanqui (1472–1493).

It's an amazing place to visit and just as impressive as the photos.
